pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>dev.snowdrop</groupId>
     <artifactId>snowdrop-dependencies</artifactId>
-    <version>2.3.0-SNAPSHOT</version>
+    <version>2.4.0-SNAPSHOT</version>
```

Comment:
Shouldn't it be `2.4.3-SNAPSHOT`?

Reply:
No, because the circleci test itself then changes that version to whatever version is being released.

https://github.com/snowdrop/spring-boot-bom/blob/facedbec5a8741be8673ec6dc591d72cff994bc7/.circleci/config.yml#L54

Reply:
ah, ah, ok then. thanks